### DISCUSSION TITLE
Add Plugin To Editor.js

### DISCUSSION THREAD

**Discussion by simon-online** [mirumee/saleor-dashboard#2141] [Q&A]
What's the best way add a plugin (eg. [Table](https://github.com/editor-js/table)) to the rich text editor Editor.js?

It looks like I'll need to add the plugin package to package.json and then include the extra config in
[src/components/RichTextEditor/consts.ts](https://github.com/saleor/saleor-dashboard/blob/main/src/components/RichTextEditor/consts.ts)

**Comment by karolkielecki** [comment]
Hi @simon-online, sorry for not getting back to you sooner. I've briefly discussed your idea on how to deal with it -- and it seems you're in the right direction. Fork the code, and then add a plugin. 

Cheers, and let us know how it works!
